src/screens/landingPageScreens/QuestionsScreen.tsx

```diff
@@ -0,0 +1,88 @@
+import React, { useState } from "react";
+
+interface CardData {
+    question: string;
+    description: string;
+}
+
+interface QuestionsScreenProps {
+    question: string;
+    description: string;
+    isOpen: boolean;
+    onClick: () => void;
+}
+
+const Questions = () => {
+    const [openCardIndex, setOpenCardIndex] = useState<number | null>(null);
+
+    const cardsData: CardData[] = [
+        {
+            question: "What countries and universities does Accademium support?",
```

Comment:
This cound be moved to a dedicated file for static data, e.g., `src/data/CardData`, and renamed to something like `questionCards: CardData[]`.